.github/ISSUE_TEMPLATE/chart_upgrade.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Chart Upgrade
+about: Request to change a Chart version
+title: ''
+labels: improvement
+assignees: ''
+
+---
+
+**What Chart should be upgraded?**
+
+/e.g., dexidp/dex 2.15/
+
+**Why should this Chart be upgraded?**
+
+- [ ] We need to perform a security upgrade.
+- [ ] We need it for a new feature: *which feature (link to blocked issue)*.
+- [ ] We need to keep up to prevent future issues.
+
+**Acceptance criteria**
+
+- I checked the migration of the new Chart:
+    - [ ] I upgraded a Chart and determined that no migration steps are needed.
+    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).
+- [ ] I tested the functionality provided by the new Chart (e.g., Auth flow, Grafana dashboards, etc.)
```

Comment:
Why do this have a checkbox? I'm betting on a miss
Also all of these are written in perspective of the implementor, I would like them more objective.
e.g. `The functionality provided by the new Chart (e.g., Auth flow, Grafana dashboards, etc.) are tested`

Reply:
I wrote "I" to make it clear that the person who takes the issue is responsible for performing these actions and should not expect QA or someone else to do this for for them. I see a risk that `The functionality provided by the new Chart (e.g., Auth flow, Grafana dashboards, etc.) are tested` creates vagueness on who should do what. Is it okay if the functionality was tested upstream? Can I rely on rumors that it worked on the demo cluster? Can I rely on someone in CNOSA having used this in a project? No, no and no! The one who picks the issue needs to put a bit more effort into ensuring no breakage.

This is my opinion. Since this is development process, @pettersv should have the final saying.

Reply:
I'm with Cristian here.